README.md

```diff
@@ -31,9 +31,16 @@ Here's how to use it in a playbook:
     - server_setup
 ```
 
+## New Repositories from a Skeleton ##
+
+Please see our [Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
+for step-by-step instructions on how to start a new repository from
+a skeleton. This will save you time and effort when configuring a
+new repository!
+
```

Comment:
We have been removing this blurb in non-skeleton repos, weren't we?

Reply:
Great catch.  Removed in commit 908cae3.